projects/tinkerbell/tinkerbell-chart/chart/values.yaml

```diff
@@ -44,13 +43,13 @@ hegel:
   roleName: hegel-role
   roleBindingName: hegel-rolebinding
   env:
-  - name: TRUSTED_PROXIES
-    value: 192.168.0.0/16
+    - name: TRUSTED_PROXIES
```

Comment:
should this be prepended with `HEGEL_`?

Reply:
Updated. It doesn't impact anything though because its overwritten in the EKS-A code. 

When we've refactored the upstream chart the overrides in EKS-A won't be necessary.